assets/xml/objects/object_ani.xml

```diff
@@ -1,61 +1,63 @@
-        <!-- <Blob Name="object_ani_Blob_006518" Size="0x380" Offset="0x6518" /> -->
-        <Texture Name="object_ani_Tex_006898" OutName="tex_006898" Format="ci8" Width="16" Height="16" Offset="0x6898" />
-        <Texture Name="object_ani_Tex_006998" OutName="tex_006998" Format="ci8" Width="8" Height="8" Offset="0x6998" />
-        <Texture Name="object_ani_Tex_0069D8" OutName="tex_0069D8" Format="ci8" Width="8" Height="8" Offset="0x69D8" />
-        <Texture Name="object_ani_Tex_006A18" OutName="tex_006A18" Format="ci8" Width="16" Height="16" Offset="0x6A18" />
-        <!-- <Blob Name="object_ani_Blob_006B18" Size="0x800" Offset="0x6B18" /> -->
-        <DList Name="object_ani_DL_007B58" Offset="0x7B58" />
-        <Animation Name="object_ani_Anim_009220" Offset="0x9220" />
-        <Animation Name="object_ani_Anim_009734" Offset="0x9734" />
-        <Animation Name="object_ani_Anim_009D34" Offset="0x9D34" />
-        <Animation Name="object_ani_Anim_00A460" Offset="0xA460" />
-        <Animation Name="object_ani_Anim_00A978" Offset="0xA978" />
-        <Animation Name="object_ani_Anim_00B2B0" Offset="0xB2B0" />
-        <Animation Name="object_ani_Anim_00B8AC" Offset="0xB8AC" />
+        <Animation Name="gAniTreeHangLosingBalanceAnim" Offset="0x7FC" />
+        <Animation Name="gAniFallOverHoldingFootAnim" Offset="0xC14" />
+        <Animation Name="gAniHoldingFootPainWrithingAnim" Offset="0x11CC" />
+        <Animation Name="gAniUnusedSidesteppingAnim" Offset="0x1584" />
+        <Animation Name="gAniTreeHangingAnim" Offset="0x1D48" />
+        <Animation Name="gAniTreeHangingReachAnim" Offset="0x27A0" />
+        <Limb Name="gAniPelvisLimb" Type="Standard" Offset="0x27B0" />
+        <Limb Name="gAniLeftThighLimb" Type="Standard" Offset="0x27BC" />
+        <Limb Name="gAniLeftCalfLimb" Type="Standard" Offset="0x27C8" />
+        <Limb Name="gAniLeftFootLimb" Type="Standard" Offset="0x27D4" />
+        <Limb Name="gAniRightThighLimb" Type="Standard" Offset="0x27E0" />
+        <Limb Name="gAniRightCalfLimb" Type="Standard" Offset="0x27EC" />
+        <Limb Name="gAniRightFootLimb" Type="Standard" Offset="0x27F8" />
+        <Limb Name="gAniTorsoLimb" Type="Standard" Offset="0x2804" />
+        <Limb Name="gAniLeftUpperArmLimb" Type="Standard" Offset="0x2810" />
+        <Limb Name="gAniLeftForArmLimb" Type="Standard" Offset="0x281C" />
+        <Limb Name="gAniLeftHandLimb" Type="Standard" Offset="0x2828" />
+        <Limb Name="gAniRightUpperArmLimb" Type="Standard" Offset="0x2834" />
+        <Limb Name="gAniRightForArmLimb" Type="Standard" Offset="0x2840" />
+        <Limb Name="gAniRightHandLimb" Type="Standard" Offset="0x284C" />
+        <Limb Name="gAniHeadLimb" Type="Standard" Offset="0x2858" />
+        <Skeleton Name="gAniSkeleton" Type="Flex" LimbType="Standard" Offset="0x28A0" />
+        <!--- most of these did not load n z64Utils, naming by matchng aganist limbs -->
```

Comment:
Comment not needed IMO. That's more a "report to Random" thing than useful for future users of the xml.

Reply:
Did you mean to delete this comment, cause I still see it.

Reply:
Oh weird... I thought I did and there is a thumbs up from me, can remove.